latte/latte.TESTGEN.ps1

```diff
@@ -2,9 +2,7 @@
 # Script Input Parameters Enforcement
 #===============================================
 Param(
-    [parameter(Mandatory=$false)] [switch] $Detail = $false,
-    [parameter(Mandatory=$false)] [Int]    $Iterations = 1,
-    [parameter(Mandatory=$false)] [ValidateSet('Sampling','Testing')] [string] $Config = "Sampling",
+    [parameter(Mandatory=$false)] [ValidateSet('Default','Azure', 'Detail')] [string] $Config = "Default",
```

Comment:
Please remove the Validation as we want to allow user to provide a custom config file